server/src/internalClusterTest/java/org/opensearch/search/SearchTimeoutIT.java

```diff
@@ -104,11 +105,10 @@ public void testSimpleTimeout() throws Exception {
     }
 
     public void testSimpleDoesNotTimeout() throws Exception {
-        final int numDocs = 10;
+        final int numDocs = 9;
```

Comment:
how does changing the numDocs help here? if the value of numDocs is changed in future to some other value, can it result in slice count<=1?

Reply:
The merge policy will trigger a merge on 10 docs so as long as it's not exactly 10 we will get multiple slices. The more important change is L110 below where we set the index refresh policy to immediate so we create a new segment per doc ingested.

> if the value of numDocs is changed in future to some other value, can it result in slice count<=1?

Yes that's possible but I think that's also possible with `indexRandomForConcurrentSearch` since we don't have a way to disable merges altogether

Reply:
got it. should we use indexRandomForConcurrentSearch rather than deciding on the number of docs to keep in the test? the worry is that someone being not aware can change the doc count and it will lose the coverage for concurrent search.

Reply:
I suppose it wouldn't hurt to add it here, will make that change

Reply:
looks like this is still pending